src/db.js

```diff
@@ -26,9 +26,22 @@ const sequelize = new Sequelize({
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || '314159',
+  logging: false,
 });
 
+const initializeDatabase = async () => {
+  try {
+    await sequelize.authenticate();
+    await sequelize.sync({ force: true });
```

Comment:
Using `force: true` will drop and recreate tables on every application start. This is useful for development but should be avoided in production to prevent data loss.